pandas/core/shared_docs.py

```diff
@@ -53,7 +53,7 @@
 ] = """
 Compare to another {klass} and show the differences.
 
-.. versionadded:: 1.1.0
+.. versionadded:: 1.5.0
```

Comment:
Please keep this as is and add another versionadded to your argument

Reply:
Done